test/articles.test.js

```diff
@@ -198,6 +208,13 @@ function checkSingleArticleProperties(article) {
     article.keywords.should.be.array
 }
 
+function checkValidError(res) {
+    res.should.have.status(500)
+    res.body.should.not.be.empty
+    res.body.should.have.property('err')
+    res.body.should.have.property('reqParams')
```

Comment:
Just a thought for the future, maybe we should change our `reqQuery` and `reqParams` to be `givens`. This way we have a consistent error message. (Currently we have both due to some routes taking params and some routes taking queries)